@xen-orchestra/lite/src/components/tasks/TasksTable.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <UiTable class="tasks-table">
+    <template #header>
+      <th>{{ $t("task.name") }}</th>
+      <th>{{ $t("task.object") }}</th>
+      <th>{{ $t("task.progress") }}</th>
+      <th>{{ $t("task.started") }}</th>
+      <th>{{ $t("task.estimated-end") }}</th>
+    </template>
+
+    <PendingTaskRow
+      v-for="task in pendingTasks"
+      :key="task.uuid"
+      :task="task"
+    />
+    <FinishedTaskRow
+      v-for="task in finishedTasks"
+      :key="task.uuid"
+      :task="task"
+    />
+  </UiTable>
+</template>
+
+<script lang="ts" setup>
+import FinishedTaskRow from "@/components/tasks/FinishedTaskRow.vue";
+import PendingTaskRow from "@/components/tasks/PendingTaskRow.vue";
+import UiTable from "@/components/ui/UiTable.vue";
+import type { XenApiTask } from "@/libs/xen-api";
+
+defineProps<{
+  pendingTasks: XenApiTask[];
+  finishedTasks: XenApiTask[];
+}>();
+</script>
+
+<style lang="postcss" scoped></style>
```

Comment:
In order to adapt to the FIGMA model:
```suggestion
<style lang="postcss" scoped>
:deep(td),
:deep(th) {
  border-right: none;
}
</style>
```

Reply:
There is indeed a difference between some tables in the Figma models 👍🏻 

Therefore, I think it would be preferable to add a prop to `UiTable` for vertical separator.

Since the majority of tables don't have the vertical separator, I'll make it an opt-in.

Reply:
I'll update UiTable on another PR to add this prop and to set the width to 100%.